librz/core/cmd/cmd_search.c

```diff
@@ -1291,39 +1329,124 @@ static void print_rop(RzCore *core, RzList /*<RzCoreAsmHit *>*/ *hitlist, PJ *pj
 			free(asm_op_hex);
 			free(buf);
 			rz_analysis_op_fini(&aop);
+			comment = NULL;
+			break;
+		case RZ_OUTPUT_MODE_TABLE:
+			buf = malloc(hit->len);
+			if (!buf) {
+				goto cleanup;
+			}
+			rz_io_read_at(core->io, hit->addr, buf, hit->len);
+			rz_asm_set_pc(core->rasm, hit->addr);
+			rz_asm_disassemble(core->rasm, &asmop, buf, hit->len);
+			rz_analysis_op_init(&aop);
+			rz_analysis_op(core->analysis, &aop, hit->addr, buf, hit->len, RZ_ANALYSIS_OP_MASK_BASIC);
+			size += hit->len;
+			if (asmop_str) {
+				asmop_str = rz_str_append(asmop_str, rz_asm_op_get_asm(&asmop));
+				const ut64 addr_last = ((RzCoreAsmHit *)rz_list_last(hitlist))->addr;
+				if (addr_last != hit->addr) {
+					asmop_str = rz_str_append(asmop_str, "; ");
+				}
+			} else {
+				asmop_str = rz_str_newf("%s; ", rz_asm_op_get_asm(&asmop));
+			}
+			char *asmop_hex_str_dup = NULL;
+			if (asmop_hex_str) {
+				asmop_hex_str_dup = rz_asm_op_get_hex(&asmop);
+				asmop_hex_str = rz_str_append(asmop_hex_str, asmop_hex_str_dup);
+			} else {
+				asmop_hex_str_dup = rz_asm_op_get_hex(&asmop);
+				asmop_hex_str = rz_str_newf("%s", asmop_hex_str_dup);
+			}
+			free(asmop_hex_str_dup);
+			// rz_table_add_rowf(state->d.t, "Xss", hit->addr, rz_asm_op_get_hex(&asmop), rz_asm_op_get_asm(&asmop));
```

Comment:
Why this is commented out?

Reply:
This needs to be removed, thanks for pointing it out